### ISSUE TITLE
1.23.1138 not installing

### ISSUE THREAD

**Issue by PhilPJL** [tom-englert/ResXResourceManager#153]
I can't get this to update

![image](https://user-images.githubusercontent.com/271656/42725933-28de1a84-8784-11e8-987c-d9132c148fb8.png)

Even if I uninstall and reinstall from scratch I end up with 1.22.1131.

**Comment by tom-englert** [comment]
Just updated and it worked fine. Did you try https://github.com/remcoros/DuplicateExtensionFinder?

**Comment by PhilPJL** [comment]
Yes I have.  No duplicates.

If I try to update the extension in VS 2017 I get
![image](https://user-images.githubusercontent.com/271656/42731855-d46b6940-880d-11e8-847b-b36b818cc8da.png)

If I uninstall and reinstall it ends up at 1.2.1131 again.

I have VS 2017 15.7.5 and VS 2017 15.8.0 preview 4 installed.

I've also tried on a different machine with just VS 2017 15.7.5.

This time I downloaded the .vsix from the marketplace.  Got the same message.  

![image](https://user-images.githubusercontent.com/271656/42731916-37c19f04-880f-11e8-95b8-d4b963a4e79f.png)

Looks like there's a packaging/version issue.  VS thinks there's an update to 1.2.1138 but .1131 is available.
